app/Models/EmailTemplate.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Models;
+
+use Spatie\MediaLibrary\HasMedia;
+use Spatie\MediaLibrary\InteractsWithMedia;
+use Illuminate\Database\Eloquent\Relations\MorphTo;
+
+class EmailTemplate extends BaseModel implements HasMedia
+{
+    use InteractsWithMedia;
+
+    protected $fillable = [
+        'name',
+        'primary_color',
+        'model_id',
+        'model_type',
+    ];
+
+    public function registerMediaCollections(): void
+    {
+        $this->addMediaCollection('logo')
+            ->singleFile();
+    }
+
+    public function model(): MorphTo
```

Comment:
I feel like the term `model` is a little to generate for what this is.

Do you have any ideas for terms that would better encapsulate this? consumer, appliedTo, etc.

Reply:
What about?
```php
public function relatedTo(): MorphTo
{
    return $this->morphTo('model');
}
```
Do you want me to to change the database columns?

Reply:
That works, yeah if you could, please.